### PR TITLE
review first part of time series tutorials

### PR DESCRIPTION
**Pull request by petrasovaa**

### CODE REVIEW

File: time_series/time_series_management_and_visualization.qmd [hunk 1]
```diff
@@ -250,7 +233,7 @@ above is one of the easiest, i.e., data represents regular intervals. So,
 
 Let's suppose however, we now work with the so called 8-day or 16-day products.
 These also represent interval time, but the last map of each year has a different
-granularity. This is because the aggregation cycle restarts every January 1st, 
+granularity. This is because the aggregation cycle restarts every January 1st,
```
Comment (petrasovaa) [inline time_series/time_series_management_and_visualization.qmd:236]: This was kind of unclear but I don't have a better wording.

File: time_series/time_series_management_and_visualization.qmd [hunk 2]
```diff
@@ -84,38 +78,29 @@ learn the basics of time series management:
...
-
-```{python}
-# Ask GRASS GIS where its Python packages are 
 import sys
 import subprocess
 
+# Ask GRASS GIS where its Python packages are
 sys.path.append(
-    subprocess.check_output([grassbin, "--config", "python_path"], text=True).strip()
+    subprocess.check_output(["/home/akratoc/dev/grass/grass_main/bin.x86_64-pc-linux-gnu/grass", "--config", "python_path"], text=True).strip()
```
Comment (veroandreo) [inline time_series/time_series_management_and_visualization.qmd:90]:
```suggestion
    subprocess.check_output(["grass", "--config", "python_path"], text=True).strip()
```

File: time_series/time_series_management_and_visualization.qmd [hunk 3]
```diff
@@ -84,38 +78,29 @@ learn the basics of time series management:
...
 # Import the GRASS GIS packages we need
 import grass.script as gs
 import grass.jupyter as gj
+```
+
+Now we are ready to start a GRASS GIS session in the downloaded project:
 
+```{python}
+path_to_project = "/data_8TB/3564515/grassdata_eu_laea_northern_italy_LST_1km_daily_celsius_reconstructed_ZLIB/eu_laea/italy_LST_daily"
+#path_to_project = "eu_laea/italy_LST_daily"
```
Comment (veroandreo) [inline time_series/time_series_management_and_visualization.qmd:101]:
```suggestion
path_to_project = "eu_laea/italy_LST_daily"
```

File: time_series/time_series_management_and_visualization.qmd [hunk 4]
```diff
@@ -369,62 +350,70 @@ print(gs.read_command("t.list",
                       where="start_time >= '2018-06-30'"))
 ```
 
+With the Python Pandas package we can simple read in the output of t.rast.list as a DataFrame:
```
Comment (veroandreo) [inline time_series/time_series_management_and_visualization.qmd:353]:
```suggestion
With the Python Pandas package we can simply read in the output of *t.rast.list* as a DataFrame:
```

**Review by veroandreo — APPROVED** [review]
Thanks @petrasovaa!

**Pull request merged**